### DISCUSSION TITLE
Sync intent supported in Orgzly Revived?

### DISCUSSION THREAD

**Discussion by rieje** [orgzly-revived/orgzly-android-revived#212] [Q&A]
In Orgzly, I have Tasker run `com.orgzly.intent.action.SYNC_START` on Syncthing launch to sync my notes. This does not seem to work for Orgzly Revived and I cannot find the counterpart to this. Is this not supported?

Thanks.

**Comment by amberin** [comment]
@rieje I'm not really familiar with this stuff, but have you tried doing it this way: https://github.com/orgzly-revived/orgzly-android-revived/discussions/144#discussioncomment-8362767 ?

Apparently that worked in February, anyway...
